test/eth.subscribe.ganache.js

```diff
@@ -35,6 +35,61 @@ describe('subscription connect/reconnect', function() {
         });
     });
 
+    it('subscribes with a callback', function(){
+        let subscription;
```

Comment:
This variable could be registered once as a global existing ``subscription`` let in the current ``describe`` closure.

Reply:
👍